extensions/import/src/wizard/pages/prosePreviewPage.ts

```diff
@@ -73,27 +101,43 @@ export class ProsePreviewPage extends ImportPage {
 
 	public setupNavigationValidator() {
 		this.instance.registerNavigationValidator((info) => {
+			if (info) {
+				// Prose Preview to Modify Columns
+				if (info.lastPage === 1 && info.newPage === 2) {
+					return !this.loading.loading && this.table.data && this.table.data.length > 0;
+				}
+			}
 			return !this.loading.loading;
 		});
 	}
 
-	private async handleProse() {
-		await this.provider.sendPROSEDiscoveryRequest({
+	private async handleProse(): Promise<boolean> {
+		return this.provider.sendPROSEDiscoveryRequest({
 			filePath: this.model.filePath,
 			tableName: this.model.table,
 			schemaName: this.model.schema,
 			fileType: this.model.fileType
 		}).then((result) => {
-			this.model.proseDataPreview = result.dataPreview;
-			this.model.proseColumns = [];
-			result.columnInfo.forEach((column) => {
-				this.model.proseColumns.push({
-					columnName: column.name,
-					dataType: column.sqlType,
-					primaryKey: false,
-					nullable: column.isNullable
-				});
-			});
+			if (result) {
+				this.model.proseDataPreview = null;
+				if (result.dataPreview) {
+					this.model.proseDataPreview = result.dataPreview;
+				}
+				this.model.proseColumns = [];
+				if (result.columnInfo) {
+					result.columnInfo.forEach((column) => {
+						this.model.proseColumns.push({
+							columnName: column.name,
+							dataType: column.sqlType,
+							primaryKey: false,
+							nullable: column.isNullable
+						});
+					});
+					return true;
```

Comment:
>return true; [](start = 5, length = 12)

this.isSuccess = true?

Reply:
Handled by the function that calls it.

Reply:
Then get rid of the this.isSuccess = false; on line 139